manual/en/creating-a-scene.html

```diff
@@ -92,7 +92,7 @@ <h2>Rendering the scene</h2>
 		<p>If you copied the code from above into the main.js file we created earlier, you wouldn't be able to see anything. This is because we're not actually rendering anything yet. For that, we need what's called a render or animation loop.</p>
 
 <pre class="prettyprint notranslate lang-js" translate="no">
-function animate() {
+function animate( time ) {
```

Comment:
Further below, there is a link to a live example (a fiddle) that needs to be updated as well to reflect the code changes.